src/frontend/komponenter/Fagsak/TilkjentYtelse/TidslinjeEtikett.tsx

```diff
@@ -7,38 +7,42 @@ import { Skalaetikett } from '@navikt/helse-frontend-tidslinje/lib/src/component
 
 interface IEtikettProp {
     etikett: Skalaetikett;
-    style: { [key: string]: string };
 }
 
-const TidslinjeEtikett: React.FunctionComponent<IEtikettProp> = ({ etikett, style }) => {
-    const { aktivEtikett, settAktivEtikett, aktivtTidslinjeVindu } = useTidslinje();
+const TidslinjeEtikett: React.FunctionComponent<IEtikettProp> = ({ etikett }) => {
+    const {
+        aktivEtikett,
+        settAktivEtikett,
+        aktivtTidslinjeVindu,
+        initiellAktivEtikettErSatt,
+        setInitiellAktivEtikettErSatt,
+    } = useTidslinje();
 
     const onEtikettClick = () => {
         settAktivEtikett(etikett);
     };
 
-    useEffect(() => {
-        if (
-            etikett.dato.getFullYear() === new Date().getFullYear() &&
-            etikett.dato.getMonth() === new Date().getMonth()
-        ) {
-            settAktivEtikett(etikett);
-        }
-    }, []);
+    if (
```

Comment:
Hvorfor endre fra useEffect? 

Hvis den skal oppdateres flere ganger burde man kanskje legge til en prop i dependency-arrayet?

Reply:
Dette skal bare skje første gang man kommer inn, og så er den ikke avhengig at ting har rendra for å sette den aktiv. 

Akkurat nå resettes den når man går til årsvisning, så da skjer det også når man går tilbake til månedsvisning. Litt usikker på om den burde da vise måneden man er i eller det siste man velget før man valgte årsvisning, men det tenker jeg ikke er så viktig å bruke tid på nå?

Reply:
Okey, skjønner 👍